signer/src/storage/postgres.rs

```diff
@@ -372,9 +417,23 @@ impl super::DbWrite for PgStore {
 
     async fn write_withdraw_request(
         &self,
-        _withdraw_request: &model::WithdrawRequest,
+        withdraw_request: &model::WithdrawRequest,
     ) -> Result<(), Self::Error> {
-        todo!(); // TODO(246): Write query + integration test
+        sqlx::query!(
+            "INSERT INTO sbtc_signer.withdraw_requests VALUES ($1, $2, $3, $4, $5, $6, $7)",
+            withdraw_request.request_id,
+            &withdraw_request.block_hash,
+            &withdraw_request.recipient,
+            withdraw_request.amount,
+            withdraw_request.max_fee,
+            withdraw_request.sender_address,
+            withdraw_request.created_at,
```

Comment:
Yeah I guess it could be useful, though I'm not sure if we do need it. We can definitely do it as a follow-up or add it whenever we find that we need it.